src/sentry/api/serializers/models/discoversavedquery.py

```diff
@@ -26,6 +26,7 @@ def serialize(self, obj, attrs, user, *args, **kwargs):
             'projects': [project.id for project in obj.projects.all()],
             'dateCreated': obj.date_created,
             'dateUpdated': obj.date_updated,
+            'createdBy': six.text_type(obj.created_by.id) if obj.created_by else None,
```

Comment:
you should be able to do `six.text_type(obj.created_by_id) if obj.created_by_id else None` instead and i believe that will save a query execution